eventserver/auth/uaa_authorizer.go

```diff
@@ -69,20 +70,36 @@ func (a ClientAuthorizer) client() (*cfclient.Client, error) {
 	})
 }
 
-func (a *ClientAuthorizer) Spaces() ([]string, error) {
+func (a *ClientAuthorizer) Organisations(requestedOrgs []string) (bool, error) {
```

Comment:
I am not really convinced that we are adding here some functionality but not really testing it, because in the tests we mock these methods. 

Maybe we should refactor the tests to actually mock the cf client itself or use a fake api server?  

With this change we add more logic than before, because we have multiple branches depending the scope (admin), if the user is a OrgManager or if the user is BillingManager. Before it was only returning the list of space IDs one to one.

Anyway, if you are OK with it, won't block this PR because this because there was not a test before.

Reply:
OK, but as said: The previous implementation was simply forwarding values, in the new  one we add a more complex behaviour. Proof of it was that the original implementation had a bug.